doc/api/cli.md

```diff
@@ -920,6 +920,16 @@ following permissions are restricted:
 * Child Process - manageable through [`--allow-child-process`][] flag
 * Worker Threads - manageable through [`--allow-worker`][] flag
 
+### `--experimental-repl`
+
+<!--
+added: REPLACME
+-->
+
+> Stability: 1 - Experimental
+
+Runs Node.js with the experimental REPL, rather than the stable implementation.
```

Comment:
Running `node --experimental-repl -e ';'` does not run the REPL, so it's not what the flag does.
```suggestion
Enables the experimental REPL..
```